team-bayou/UNO-Shareboard
# Enhancement/#9/add user

closes #9 
This enhancement adds a complete data bridge from controller to repository.

To Test: 
-Checkout the branch enhancement/#9/add_user
-Open up postman
-Click on Import
-Click choose file and navigate to the postman collection to import under resources/postman in the project
-Once imported, open up the project in your IDE and run it locally
-Go back to postman and press SEND to test the addUser controller 
-The response back should be a 200OK and json body with the string "jleaton"
